src/transactions/reducer.ts

```diff
@@ -46,7 +46,7 @@ export interface KnownFeedTransactionsType {
   [txHash: string]: string | boolean
 }
 
-const initialState = {
+export const initialState = {
```

Comment:
nit: Also this is only exported for testing, we can either use our convention of using `export const _initialState = initialState` or  calling `reducer(undefined, { type: 'does-not-matter' })` to get the same.

To avoid exporting things that shouldn't be exported.